R/pkg/DESCRIPTION

```diff
@@ -62,3 +62,4 @@ Collate:
 RoxygenNote: 5.0.1
 VignetteBuilder: knitr
 NeedsCompilation: no
+Encoding: UTF-8
```

Comment:
This one, we had to revert before due to Jekins enviornment WIP. I think now it's all good to go.